datalake/src/main/java/com/sequenceiq/datalake/service/sdx/attach/SdxDetachService.java

```diff
@@ -69,7 +76,8 @@ public SdxCluster detachCluster(Long sdxID) {
      */
     public void detachStack(SdxCluster cluster, String originalName) throws Exception {
         LOGGER.info("Started detaching stack of SDX cluster with ID: {}.", cluster.getId());
-        sdxAttachDetachUtils.updateStack(cluster, originalName);
+        sdxAttachDetachUtils.updateStack(originalName, cluster.getClusterName(), cluster.getCrn());
+        jobService.schedule(cluster.getId(), SdxClusterJobAdapter.class);
```

Comment:
What exactly is this jobservice used for?

Reply:
This schedules an SDX status-checked task to poll the status of the cluster resources and update the cluster state accordingly.